rqt_gui_cpp/include/rqt_gui_cpp/plugin.h

```diff
@@ -37,18 +37,16 @@
 #include <qt_gui_cpp/plugin_context.h>
 #include <qt_gui_cpp/settings.h>
 
-#include <nodelet/nodelet.h>
+#include <rclcpp/rclcpp.hpp>
 
 namespace rqt_gui_cpp {
 
 /**
  * The base class for C++ plugins which use the ROS client library.
- * A plugin must not call ros::init() as this is performed once by the framework.
- * The name of the ROS node consists of the prefix "rqt_gui_cpp_node_" and the process id.
+ * A plugin must not call rclcpp::init() as this is performed once by the framework.
  */
 class Plugin
   : public qt_gui_cpp::Plugin
-  , public nodelet::Nodelet
```

Comment:
With the removal of the base class what is the replacement? A node handle provided through the context?

Reply:
This PR is only for getting things to compile correctly